src/lib.rs

```diff
@@ -28,6 +28,7 @@ pub struct Category {
 #[derive(Deserialize, Debug)]
 pub struct Channel {
     #[serde(rename = "type")]
+    pub channel_id: u64,
```

Comment:
This struct gets deserialized from JSON when building the database, but the JSON files don't contain a `channel_id` so we need to skip it. Also the rename should stay on the `channel_type` field:

```suggestion
    #[serde(skip)]
    pub channel_id: u64,
    #[serde(rename = "type")]
```